houstory/router.js

```diff
@@ -23,7 +23,7 @@ router.get('/', jsonParser, (req, res) => {
   //from MyLibrary/library/router.js
   let userId = req.user.id;
   return Houstory.find({userId})
-  .then(posts.map(post.serialize()));
```

Comment:
This way tries to evaluate the method calls using `posts` and `post` immediately, instead of chaining them after the `find`.